src/components/RecomendationCard/RecomendationCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { Paragraph } from '@/index';
+import { IRecomendationCard, IRecomendationCardFooter, IRecomendationCardImage } from './RecomendationCardTypes';
+import ImageEmpty from '@/images/ImageEmpty';
+
+const RecomendationCard = ({ src, title, price }: IRecomendationCard) => {
+
+  return (
+    <div
+      className={`rounded-[4px] border-solid sm:border-[1px] border-dark-300 max-w-[153px] sm:max-w-[392px] flex flex-col items-start`}
+    >
+      <RecomendationCard.Image src={src}/>
+      <RecomendationCard.Footer title={title} price={price}/>
+    </div>
+  );
+};
```

Comment:
The selected state missing, let's see where the guys apply it and check if we need it 
![image](https://user-images.githubusercontent.com/105232333/228313988-a4334681-bbaa-4cee-a994-f3ac7246e00c.png)